app/assets/javascripts/discourse/templates/components/date-time-input-range.hbs

```diff
@@ -1,35 +1,16 @@
-<ul class="panels {{currentPanel}}">
-  <li>
-    {{d-button
-      label="date_time_picker.from"
-      class="from-panel"
-      action=(action "onChangePanel" "from")}}
-  </li>
-  <li>
-    {{d-button
-      label="date_time_picker.to"
-      class="to-panel"
-      action=(action "onChangePanel" "to")}}
-  </li>
-</ul>
+{{date-time-input
+  date=from
+  onChange=(action "_onChange" (hash prop="from"))
+  showTime=showFromTime
+  class="from"
+}}
 
-{{#if error}}
-  <div class="alert error">{{error}}</div>
-{{/if}}
-
-<div class="panel from {{if fromPanelActive "visible"}}">
-  {{date-time-input
-    date=from
-    onChange=(action "_onChange" (hash prop="from"))
-    showTime=showFromTime
-  }}
-</div>
-
-<div class="panel to {{if toPanelActive "visible"}}">
-  {{date-time-input
-    date=to
-    onChange=(action "_onChange" (hash prop="to"))
-    showTime=showToTime
-    clearable=true
-  }}
-</div>
+{{date-time-input
+  date=to
+  relativeDate=from
+  onChange=(action "_onChange" (hash prop="to"))
```

Comment:
This is a bit unusual. An underscore implies it's a private API yet it's used in the template? I recommend a different name.